src/guide/jbs-jdk-bug-system.md

```diff
@@ -243,6 +243,12 @@ As mentioned above, issues that lack the information needed to investigate the p
 
 Once a bug is marked as fixed there is now the option of someone, other than the person that fixed it, of marking it as [Verified]{.jbs-value} to confirm that the issue is fixed after testing; marking it as [Fix Failed]{.jbs-value} if it didn't solve the issue; or, [Not Verified]{.jbs-value} to indicate that it wasn't explicitly tested. Note that the JBS UI doesn't highlight when [Fix Failed]{.jbs-value} has been set, you need to look for the [Verification]{.jbs-field} field at the bottom of the left-hand column in the Details section.
 
+## Removing an issue
+
+Removing a JBS issue is a rare extreme case that shouldn't be part of the normal workflow. For this reason, removing issues is restricted to admins only. If you for some reason need to remove an issue, send an email to [ops@openjdk.org](mailto:ops@openjdk.org). You need to provide the bug id and a well thought through reason to why the issue should be removed.
+
+Note that JBS issues are not removed just because something was a bad idea, or reported an issue that turned out to be an embarassing user misstake. Such issues are simply closed.
```

Comment:
```suggestion
Note that JBS issues are not removed just because something was a bad idea, or reported an issue that turned out to be an embarrassing user mistake. Such issues are simply closed.
```

Reply:
Good catch!  That was an embarrassing mistake. :-)